docs/configuration/indexes.md

```diff
@@ -211,6 +211,26 @@ credentials cannot be found. If the discovered credentials are not valid (i.e.,
 HTTP 401 or 403), then uv will treat packages as unavailable and query the next configured index as
 described in the [index strategy](#searching-across-multiple-indexes) section.
 
+### Ignoring error codes when searching across indexes
+
+When using the [first-index strategy](#searching-across-multiple-indexes), uv will stop searching if
+an HTTP 401 Unauthorized or HTTP 403 Forbidden status code is encountered. The one exception is that
```

Comment:
What does "stop searching" mean, for a user? Similarly, what does it mean to "keep" searching?

Reply:
I thought it was clear from the section title, but I'll make it more explicit.